src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/model/in/OcrDataValidationRequest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.model.in;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.OcrDataField;
+
+import java.util.List;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+public class OcrDataValidationRequest {
+
+    @ApiModelProperty(value = "List of ocr data fields to be validated.", required = true)
+    @NotEmpty
+    public final List<OcrDataField> ocrDataFields;
+
+    @ApiModelProperty(value = "Type of the paper based form.", required = true)
+    @NotBlank
+    public final String formType;
```

Comment:
can we make this enum as form subtypes will be predefined and known ?

Reply:
sure, I'll change it to enum it in the next PR which have OCR form changes